bin/reset-local-dev

```diff
@@ -15,5 +15,7 @@ helm install \
   -n airflow \
   --set executor="$EXECUTOR" \
   --set workers.persistence.fixPermissions=true \
+  --set dagServer.enabled=True \
```

Comment:
We need to delete line 18 before merging. Line 19 is fine to keep.

Reply:
removed dagServer.enabled